workflow/gatk/mutect2_pon.wdl

```diff
@@ -0,0 +1,163 @@
+
+    # runtime
+    String gatk_docker
+    File? gatk_override
+    String basic_bash_docker = "ubuntu:16.04"
+
+    Int preemptible = 2
+    Int max_retries = 2
+    Int small_task_cpu = 2
+    Int small_task_mem = 4
+    Int small_task_disk = 100
+    Int boot_disk_size = 12
+  }
+
+  Runtime standard_runtime = {"gatk_docker": gatk_docker, "gatk_override": gatk_override,
+            "max_retries": max_retries, "preemptible": preemptible, "cpu": small_task_cpu,
+            "machine_mem": small_task_mem * 1000, "command_mem": small_task_mem * 1000 - 500,
+            "disk": small_task_disk, "boot_disk_size": boot_disk_size}
+
+    scatter (normal_bam in zip(normal_bams, normal_bais)) {
+        call m2.Mutect2 {
+            input:
+                intervals = intervals,
+                ref_fasta = ref_fasta,
+                ref_fai = ref_fai,
+                ref_dict = ref_dict,
+                tumor_reads = normal_bam.left,
+                tumor_reads_index = normal_bam.right,
+                scatter_count = scatter_count,
+                m2_extra_args = select_first([m2_extra_args, ""]) + "--max-mnp-distance 0",
+                gatk_override = gatk_override,
+                gatk_docker = gatk_docker,
+                preemptible = preemptible,
+                max_retries = max_retries,
+                gcs_project_for_requester_pays = gcs_project_for_requester_pays
+        }
+    }
+
+    call m2.SplitIntervals {
+        input:
+            ref_fasta = ref_fasta,
+            ref_fai = ref_fai,
+            ref_dict = ref_dict,
+            scatter_count = create_panel_scatter_count,
+            split_intervals_extra_args = "--dont-mix-contigs --min-contig-size " + min_contig_size,
+            runtime_params = standard_runtime
+    }
+
+    scatter (subintervals in SplitIntervals.interval_files ) {
+            call CreatePanel {
+                input:
+                    input_vcfs = Mutect2.filtered_vcf,
+                    intervals = subintervals,
+                    ref_fasta = ref_fasta,
+                    ref_fai = ref_fai,
+                    ref_dict = ref_dict,
+                    gnomad = gnomad,
+                    gnomad_idx = gnomad_idx,
+                    output_vcf_name = pon_name,
+                    create_pon_extra_args = create_pon_extra_args,
+                    runtime_params = standard_runtime
+            }
+    }
+
+    call m2.MergeVCFs {
+        input:
+            input_vcfs = CreatePanel.output_vcf,
+            input_vcf_indices = CreatePanel.output_vcf_index,
+            compress_vcfs = compress,
+            runtime_params = standard_runtime
+    }
+
+    output {
+        File pon = MergeVCFs.merged_vcf
+        File pon_idx = MergeVCFs.merged_vcf_idx
+        Array[File] normal_calls = Mutect2.filtered_vcf
+        Array[File] normal_calls_idx = Mutect2.filtered_vcf_idx
+    }
+}
+
+task CreatePanel {
+    input {
+      File intervals
+      Array[String] input_vcfs
```

Comment:
The command generation below is assuming a non-empty list, you could annotate that directly here with the `+` [non-empty postfix quantifier](https://github.com/openwdl/wdl/blob/main/versions/1.1/SPEC.md#:~:text=the%20non%2Dempty%20postfix%20quantifier)

```suggestion
      Array[String]+ input_vcfs
```